include/interaction/Simulation.h

```diff
@@ -0,0 +1,98 @@
+#ifndef SIMULATION_H
+#define SIMULATION_H
+
+#include "choreograph/Choreograph.h"
+
+#include <glm/glm.hpp>
+
+#include "PointOfInterests.h"
+
+enum ANIMATION_STATE
+{
+  STOP,
+  PLAYING,
+  PAUSE
+};
+
+/*
+  SimulationState represents a frame in the simulation/animation
+  where the camera is set on a postion P, pointing on direction of a target T,
+  and min/max are set on a X-Y value.
+  the animation change SimulationStates by interpolating between the mentioned values.
+*/
+
+#define SIMULATION_TIME_STEP 30.0
+
+struct SimulationState
+{
+  SimulationState(){};
+
+  SimulationState(const SimulationState &other)
+  {
+    poi = other.poi;
+    max_clip = other.max_clip;
+    min_clip = other.min_clip;
+  }
+
+  std::string time_label;
+  PointOfInterest poi;
+  glm::vec3 max_clip;
+  glm::vec3 min_clip;
+};
+
+class VRVolumeApp;
+
+class Simulation
+{
+public:
+  Simulation(VRVolumeApp &controller_app, float time = 10.0f);
```

Comment:
This file could use a run through the linter

Reply:
I re run it, nothing much changed